mysql-test/suite/rdrs/include/run_rdrs_unit_test.inc

```diff
@@ -30,7 +30,7 @@ if (!$res)
 }
 
 
-my $cmd="LD_LIBRARY_PATH=$ENV{LD_LIBRARY_PATH}; CGO_LDFLAGS=$ENV{CGO_LDFLAGS} $rdrs_test_cmd";
+my $cmd="export DYLD_LIBRARY_PATH=$ENV{LD_LIBRARY_PATH}; export LD_LIBRARY_PATH=$ENV{LD_LIBRARY_PATH}; export CGO_LDFLAGS=$ENV{CGO_LDFLAGS}; $rdrs_test_cmd";
```

Comment:
DYLDB_LIBRARY_PATH=LD_LIBRARY_PATH in first export, need fix

Reply:
this is correct